src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,18 @@
 package mate.academy.hibernate.relations.model;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+
+@Entity
 public class Actor implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String name;
+    @OneToOne
```

Comment:
One country can have a maximum of 1 actor? I guess their salaries must be huge if we have only around 200 of them 😄 
How many actors can 1 county have?
How many home counties can 1 actor have?
What would be a relation between them, judjing by the answers to those 2 last questions? always ask them yourself if you are not sure.

We also need to specify the `JoinColumn`/`JoinTable` depending on what relation you have, that goes for other entities as well

Reply:
I explain this for my self like a one actor can have only one country, but thx for hint